.gitignore

```diff
@@ -8,3 +8,4 @@ eggs/
 dist/
 
 venv/
+.idea/
```

Comment:
Since this is not opera-specific ignore, it would make more sense to add it into `$XDG_CONFIG/git/ignore` where all user-specific ignores should be (like editor backup buffers, config folders, etc.). As a bonus, you only need to add it once and it will be ignored in all of your git repos.

Reply:
I never considered to use global git configuration, but in this case seem to be the best choice, thank you.